lib/attrsets.nix

```
@@ -1075,10 +1075,10 @@ rec {
 
      Example:
        getOutput "dev" pkgs.openssl
-       => "/nix/store/9rz8gxhzf8sw4kf2j2f1grr49w8zx5vj-openssl-1.0.1r-dev"
+       => <<derivation /nix/store/9rz8gxhzf8sw4kf2j2f1grr49w8zx5vj-openssl-1.0.1r-dev>>
```

The issue is that while the returned value is technically a derivation, it is also an attribute set with `outputName` and `outPath` attributes changed, compared to the original package.

So while the following it technically correct output, it is useless since it will be the same for all outputs:

```suggestion
       => «derivation /nix/store/1c40mjrwhm1nmq0xa539mlsvmrd90l81-openssl-3.0.12.drv»
```

Maybe we should instead change the example code to have it stringify the attribute set:

```nix
"${lib.getOutput "dev" pkgs.openssl}"
```

---

Pushed the review changes.